src/main/groovy/util/StringUtil.groovy

```diff
@@ -52,5 +53,9 @@ class StringUtil {
         return null
     }
 
+    static String removeWhiteSpace(String s) {
```

Comment:
Why not `StringUtils.deleteWhitespace` or `StringUtils.deleteSpace`?

Reply:
I went with @EWaterman suggestion